### ISSUE TITLE
Issue with non-window dialogs - ref #14....

### ISSUE THREAD

**Issue by MHacker9404** [FantasticFiasco/mvvm-dialogs#17]
I've looked at your samples for this - but all of your samples still use Windows- based dialogs.  Do you have a sample using a UserControl as the basis for the dialog?

**Comment by FantasticFiasco** [comment]
### TL;DR
No

### Longer answer
Let me start with context. I am in the following argument referring to the demo sample called `Demo.ModalCustomDialog`.

In that sample I display that any implementation of `IWindow` can be used in combination with MVVM Dialogs. `IWindow` has no dependencies to Windows dialogs, but the implementation of `IWindow`, in this specific sample, is using the ordinary `System.Windows.Window` class. This however is not exposed to MVVM Dialogs, proving that the framework support custom implementations of dialogs.

If you are looking for implementations of windows that aren't implementations of `System.Windows.Window`, I think Telerik in their [UI for WPF](http://www.telerik.com/products/wpf/overview.aspx) is basing their implementation of a window on `System.Windows.Controls.Control` or something similar, if I remember correct. But I could be wrong.

**Comment by FantasticFiasco** [comment]
Do you still consider this to be an issue or can I close it?

**Comment by MHacker9404** [comment]
You can close it - thanks
